test/framework/alertmanager.go

```diff
@@ -290,10 +293,14 @@ func (f *Framework) WaitForAlertmanagerConfigToContainString(ns, amName string,
 			return true, nil
 		}
 
-		log.Printf("\n\nExpected:\n\n%#+v\n\nto contain:\n\n%#+v\n\n", config.Data.ConfigYAML, expectedString)
-
 		return false, nil
 	})
+
+	if err != nil {
+		return fmt.Errorf("failed to wait for alertmanager config to contain '%v': %v: %v", expectedString, err, pollError)
```

Comment:
should we use `%q` instead of `'%v'` to format the `expectedString`?

Reply:
Oh cool, wasn't aware of that.